src/components/screens/login/networkSelector/customNode/editMode.js

```diff
@@ -0,0 +1,125 @@
+import React, { useEffect, useRef, useState } from 'react';
+
+import { tokenMap } from '@constants';
+import { getNetworkConfig } from '@api/network';
+import { PrimaryButton } from '@toolbox/buttons';
+import { Input } from '@toolbox/inputs';
+import { addHttp } from '@utils/login';
+import styles from '../networkSelector.css';
+
+const validateNode = async (address) => {
+  try {
+    const response = await getNetworkConfig({
+      name: 'customNode',
+      address,
+    }, tokenMap.LSK.key);
+    if (response) {
+      return true;
+    }
+    throw new Error('error connecting');
```

Comment:
```suggestion
    throw new Error(`Failed to return response for custom node url: ${address}`);
```

Reply:
This error is not displayed. The actual error message displayed is [here](https://github.com/LiskHQ/lisk-desktop/blob/3762-store-custom-node-url/src/components/screens/login/networkSelector/customNode/editMode.js#L56).